<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBooksDialog</class>
 <widget class="QDialog" name="FindBooksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLineEdit" name="author_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="min_pages_spinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_pages_spinBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>251</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Choose parameters to find books</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="author_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author's name and surname</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hard_covers_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Have hard covers</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="illustrations_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Have Illustrations</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="year_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Year of publishment</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isbn_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pages_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of pages</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="title_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>97</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="year_lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>dfgh</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="placeholderText">
    <string>Year of publishment</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="edition_size_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edition size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>≤ pages ≤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="min_edition_size_spinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="max_edition_size_spinBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>≤ edition size ≤</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="isbn_lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="placeholderText">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="yes_illustrations_radioButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="yes_hard_covers_radioButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="no_illustrations_radioButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="no_hard_covers_radioButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Find books with specified parameters</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="author_surname_lineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Surname</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
